addons/Media_Recorder/src/presenter.js

```diff
@@ -388,6 +390,8 @@ var _AudioPlayer = __webpack_require__(37);
 
 var _DefaultRecordingPlayButton = __webpack_require__(40);
 
+var _upgradeModel = __webpack_require__(41);
```

Comment:
Czy to powinno wejść do produkcji kod, który jest generowany automatycznie? Czy na pewno to tak powinno działać?